sympy/physics/control/lti.py

```diff
@@ -1422,3 +1445,426 @@ def _eval_rewrite_as_TransferFunction(self, num, den, **kwargs):
+    [------------    ----------  ]
+    [   p + s         2          ]
+    [                s  + s + 1  ]
+    [                            ]
+    [                 4          ]
+    [     3        - p  + 3*p - 2]
+    [   -----      --------------]
+    [   s + 2          p + s     ]
+
+    ``subs()`` also supports multiple substitutions.
+
+    >>> tfm_2.subs({p: 2, a: 1})  # substituting p with 2 and a with 1
+    TransferFunctionMatrix(((TransferFunction(s + 1, s**2 + s + 1, s), TransferFunction(-3, s + 2, s)), (TransferFunction(12, s + 2, s), TransferFunction(-s - 1, s**2 + s + 1, s)), (TransferFunction(3, s + 2, s), TransferFunction(-12, s + 2, s))))
+    >>> pprint(_, use_unicode=False)
+    [  s + 1        -3     ]
+    [----------    -----   ]
+    [ 2            s + 2   ]
+    [s  + s + 1            ]
+    [                      ]
+    [    12        -s - 1  ]
+    [  -----     ----------]
+    [  s + 2      2        ]
+    [            s  + s + 1]
+    [                      ]
+    [    3          -12    ]
+    [  -----       -----   ]
+    [  s + 2       s + 2   ]
+
+    See Also
+    ========
+
+    TransferFunction, Series, Parallel, Feedback
+
+    """
+    def __new__(cls, arg):
+
+        expr_mat_arg = []
+        var = arg[0][0].var
+        for row in range(len(arg)):
+            temp = []
+            for col in range(len(arg[row])):
+                if not isinstance(arg[row][col], (TransferFunction, Series, Parallel)):
+                    raise TypeError("Incompatible type found as the element of TransferFunctionMatrix")
+
+                if var != arg[row][col].var:
+                    raise ValueError("Conflicting value(s) found for `var`. All TransferFunction instances in "
+                            "TransferFunctionMatrix should use the same complex variable in Laplace domain.")
+
+                if isinstance(arg[row][col], TransferFunction):
+                    temp.append(arg[row][col].to_expr())
+                else:
+                    temp.append(arg[row][col]._to_expr())
+            expr_mat_arg.append(temp)
+
+
+        if isinstance(arg, (list, Tuple)):
+            # Making nested Tuple (sympy.core.containers.Tuple) from nested list or nested python tuple
+            arg = Tuple(*(Tuple(*r, sympify=False) for r in arg), sympify=False)
+
+        obj = super(TransferFunctionMatrix, cls).__new__(cls, arg)
+        obj._expr_mat = ImmutableMatrix(expr_mat_arg)
+        return obj
+
+    @classmethod
+    def from_matrix(cls, matrix, var):
+        """
+        Creates a new ``TransferFunctionMatrix`` efficiently from a SymPy Matrix of ``Expr`` objects.
+
+        Parameters
+        ==========
+
+        matrix : ``ImmutableMatrix`` having ``Expr``/``Number`` elements.
+        var : Symbol
+            Complex variable of the Laplace transform which will be used by the
+            all the ``TransferFunction`` objects in the ``TransferFunctionMatrix``
+            object.
```

Comment:
```suggestion
            A complex variable of the Laplace transform which will be used by
            all the ``TransferFunction`` objects in ``TransferFunctionMatrix``.
```
Avoid writing _object(s)_ everywhere.